docs/exploration-and-organization/history.md

```diff
@@ -0,0 +1,80 @@
+---
+title: History
+---
+
+# History
+
+## Viewing tracked changes
+
+1. Go to your question, dashboard, or model.
+2. Click the info icon.
+3. A sidebar will pop up with a history of up to 15 versions.
+
+Metabase will keep track of a version each time you [save](../questions/sharing/answers.md#how-to-save-a-question), [move](../questions/sharing/answers.md#editing-your-question), [revert](#reverting-to-previous-versions), [archive](#archive), or [verify](./exploration.md#verified-items) a question or model.
+
+## Reverting to previous versions
+
+1. Go to your question, dashboard, or model.
+2. Click the info icon.
+3. A sidebar will appear with up to 15 previous versions.
+4. Click on the **back arrow** icon beside a version to revert your item to that point in time.
+
+## Archive
+
+Sometimes your questions or dashboards outlive their usefulness and need to be sent to Metabase Heaven.
```

Comment:
```suggestion
Sometimes your questions or dashboards outlive their usefulness and need to be sent to the Metabase Archives.
```
Super annoying note from me here: I like the joke, but it's off/imprecise. I'm told that people generally can't return from heaven.  Feel free to rephrase with another term.

Reply:
No problem, it's not my joke 😆 Just wanted to preserve a bit of personality from the original doc.